google/cloud/storage/acl.py

```diff
@@ -534,9 +532,7 @@ def _save(
 
         self.loaded = True
 
-    def save(
```

Comment:
Was this change already included in a published release? If so, we *might* prefer to change retry to default to None rather than removing it, in the case that a customer is already using the new signature.

Likewise for all other public methods.

If it hasn't been included in a release then we're in the clear.

Reply:
No, it wasn't published (merged yesterday, in fact).